azure-spring-boot/src/main/resources/serviceEndpoints.properties

```diff
@@ -4,6 +4,8 @@ azure.service.endpoints.cn.aadKeyDiscoveryUri=https://login.partner.microsoftonl
 azure.service.endpoints.cn.aadMembershipRestUri=https://graph.chinacloudapi.cn/me/memberOf?api-version=1.6
 azure.service.endpoints.global.aadSigninUri=https://login.microsoftonline.com/
 azure.service.endpoints.global.aadGraphApiUri=https://graph.windows.net/
+azure.service.endpoints.global.aadGraphApiUriNew=https://graph.microsoft.com/
```

Comment:
Is it possible we have two `serviceEndpoints.properties` file? e.g., `aadGraphServiceEndpoints.properties` and `microsoftGraphServiceEndpoints.properties`? Each of them have the same key list, instead of mixing them in one file?

Reply:
so i went ahead and kept the serviceEndpoints.properties for the aad graph, and then microsoftGraphServiceEndpoints.properties for the microsoft graph api.  in this file, i set the environment as global-microsoftgraph.  So a property is of the form: azure.service.endpoints.global-microsoftgraph.aadMembershipRestUri.

let me know if this is acceptable or you had envisioned a different approach.